OrbitService/main.cpp

```diff
@@ -38,7 +38,9 @@ void InstallSigintHandler() {
 }  // namespace
 
 int main(int argc, char** argv) {
-  const std::string log_file_path = Path::GetServiceLogFilePath();
+  std::filesystem::path var_log{"/var/log"};
+  std::filesystem::create_directory(var_log);
+  const std::string log_file_path = var_log / "OrbitService.log";
```

Comment:
also maybe preserve a function here just make it static?

Reply:
just to declutter the main method? Okay, will do

Reply:
done